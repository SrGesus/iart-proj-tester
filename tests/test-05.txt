FC	VC	VC
VC	BB	LH
FE	VB	FD