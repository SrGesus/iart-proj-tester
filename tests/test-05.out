FB	VB	VE
VD	BE	LV
FD	VC	FC
